code/build/template/data/config/ExtensionTemplate.ini

```diff
@@ -192,7 +192,12 @@ CheatCodeOrPhrase       = [Command]; NB: Cheat codes/phrases are case insensitiv
 [+inspector
 
 [ObjectTemplate]
-InspectorFields         = Section1.Name1 # ... # SectionN.NameN; NB: Defines additional config properties to monitor when inspecting this object;]
+InspectorFields         = Section1.Name1 # ... # SectionN.NameN; NB: Defines additional config properties to monitor when inspecting this object;
+TriggerList             = InspectorTriggerTemplate # TriggerTemplate1 # ... # TriggerTemplateN; NB: Defines handlers for inspector trigger events;
+
+[InspectorTriggerTemplate]

```

Comment:
Would you prefer for this to be named like it is, with the `Inspector` prefix to indicate that these events are specific to the inspector extension? Or name the section `TriggerTemplate` as an indication that this is the same as any other trigger definitions, except a few new ones are defined/available?

Reply:
Given `TriggerTemplate` is already defined in `CreationTemplate.ini` I'd simply go with that name and remove the `TriggerList` line altogether.

Reply:
Updated in e83b008!